<?xml version="1.0" encoding="utf-8"?>
<?xml-stylesheet type="text/xsl" href="c:\program files (x86)\microsoft visual studio 12.0\team tools\static analysis tools\fxcop\Xml\CodeAnalysisReport.xsl"?>
<FxCopReport Version="12.0">
 <Targets>
  <Target Name="C:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades\bin\Release\Entidades.dll">
   <Modules>
    <Module Name="entidades.dll">
     <Namespaces>
      <Namespace Name="Entidades">
       <Types>
        <Type Name="BloqueoUsuarios" Kind="Class" Accessibility="Public" ExternallyVisible="True">
         <Members>
          <Member Name="#Eliminar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="BloqueoUsuarios.cs" Line="90">'BloqueoUsuarios.Eliminar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Guardar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="BloqueoUsuarios.cs" Line="54">'BloqueoUsuarios.Guardar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Obtener()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="BloqueoUsuarios.cs" Line="140">'BloqueoUsuarios.Obtener()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ValidarPorId()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="BloqueoUsuarios.cs" Line="190">'BloqueoUsuarios.ValidarPorId()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
         </Members>
        </Type>
        <Type Name="Directorios" Kind="Class" Accessibility="Public" ExternallyVisible="True">
         <Members>
          <Member Name="#Eliminar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Directorios.cs" Line="348">'Directorios.Eliminar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Guardar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Directorios.cs" Line="321">'Directorios.Guardar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListadoBasicoReporte()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Directorios.cs" Line="203">'Directorios.ObtenerListadoBasicoReporte()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Directorios.cs" Line="215">'Directorios.ObtenerListadoBasicoReporte()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListadoCompletoReporte()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Directorios.cs" Line="249">'Directorios.ObtenerListadoCompletoReporte()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Directorios.cs" Line="261">'Directorios.ObtenerListadoCompletoReporte()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListadoPorId()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Directorios.cs" Line="157">'Directorios.ObtenerListadoPorId()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Directorios.cs" Line="169">'Directorios.ObtenerListadoPorId()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerPredeterminado()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Directorios.cs" Line="97">'Directorios.ObtenerPredeterminado()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Directorios.cs" Line="109">'Directorios.ObtenerPredeterminado()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Predeterminar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Directorios.cs" Line="290">'Directorios.Predeterminar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
         </Members>
        </Type>
        <Type Name="Estilos" Kind="Class" Accessibility="Public" ExternallyVisible="True">
         <Members>
          <Member Name="#ObtenerListado()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Estilos.cs" Line="66">'Estilos.ObtenerListado()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
         </Members>
        </Type>
        <Type Name="Licencia" Kind="Class" Accessibility="Public" ExternallyVisible="True">
         <Members>
          <Member Name="#Editar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Licencia.cs" Line="109">'Licencia.Editar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Eliminar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Licencia.cs" Line="132">'Licencia.Eliminar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Guardar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Licencia.cs" Line="83">'Licencia.Guardar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListado()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Licencia.cs" Line="57">'Licencia.ObtenerListado()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
         </Members>
        </Type>
        <Type Name="Modulos" Kind="Class" Accessibility="Public" ExternallyVisible="True">
         <Members>
          <Member Name="#Editar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Modulos.cs" Line="239">'Modulos.Editar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Eliminar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Modulos.cs" Line="266">'Modulos.Eliminar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Guardar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Modulos.cs" Line="213">'Modulos.Guardar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListado()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Modulos.cs" Line="56">'Modulos.ObtenerListado()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListadoPorId()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Modulos.cs" Line="91">'Modulos.ObtenerListadoPorId()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListadoReporte()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Modulos.cs" Line="153">'Modulos.ObtenerListadoReporte()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerMenuListado()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Modulos.cs" Line="127">'Modulos.ObtenerMenuListado()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ValidarPorId()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Modulos.cs" Line="187">'Modulos.ValidarPorId()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
         </Members>
        </Type>
        <Type Name="Programas" Kind="Class" Accessibility="Public" ExternallyVisible="True">
         <Members>
          <Member Name="#Editar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Programas.cs" Line="177">'Programas.Editar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Eliminar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Programas.cs" Line="211">'Programas.Eliminar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Guardar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Programas.cs" Line="151">'Programas.Guardar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListado()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Programas.cs" Line="56">'Programas.ObtenerListado()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListadoReporte()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Programas.cs" Line="125">'Programas.ObtenerListadoReporte()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerMenuListado()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Programas.cs" Line="93">'Programas.ObtenerMenuListado()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ValidarPorId()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Programas.cs" Line="255">'Programas.ValidarPorId()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
         </Members>
        </Type>
        <Type Name="Registros" Kind="Class" Accessibility="Public" ExternallyVisible="True">
         <Members>
          <Member Name="#Editar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Registros.cs" Line="71">'Registros.Editar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#EditarSoloSesion()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Registros.cs" Line="95">'Registros.EditarSoloSesion()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Eliminar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Registros.cs" Line="118">'Registros.Eliminar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Guardar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Registros.cs" Line="46">'Registros.Guardar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListadoPorNombreEquipo()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Registros.cs" Line="220">'Registros.ObtenerListadoPorNombreEquipo()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ValidarPorNombreEquipo()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Registros.cs" Line="153">'Registros.ValidarPorNombreEquipo()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ValidarSesionPorNombreEquipo()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Registros.cs" Line="184">'Registros.ValidarSesionPorNombreEquipo()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
         </Members>
        </Type>
        <Type Name="SubProgramas" Kind="Class" Accessibility="Public" ExternallyVisible="True">
         <Members>
          <Member Name="#Editar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="SubProgramas.cs" Line="233">'SubProgramas.Editar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Eliminar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="SubProgramas.cs" Line="266">'SubProgramas.Eliminar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Guardar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="SubProgramas.cs" Line="206">'SubProgramas.Guardar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListadoReporte()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="SubProgramas.cs" Line="130">'SubProgramas.ObtenerListadoReporte()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerMenuListado()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="SubProgramas.cs" Line="66">'SubProgramas.ObtenerMenuListado()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ValidarPorId()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="SubProgramas.cs" Line="179">'SubProgramas.ValidarPorId()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ValidarPorIdModuloyIdPrograma()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="SubProgramas.cs" Line="97">'SubProgramas.ValidarPorIdModuloyIdPrograma()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
         </Members>
        </Type>
        <Type Name="Usuarios" Kind="Class" Accessibility="Public" ExternallyVisible="True">
         <Members>
          <Member Name="#Editar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Usuarios.cs" Line="89">'Usuarios.Editar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Eliminar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Usuarios.cs" Line="118">'Usuarios.Eliminar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#Guardar()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Usuarios.cs" Line="61">'Usuarios.Guardar()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListado()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Usuarios.cs" Line="225">'Usuarios.ObtenerListado()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListadoPorId()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Usuarios.cs" Line="262">'Usuarios.ObtenerListadoPorId()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ObtenerListadoPorNombre()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Usuarios.cs" Line="155">'Usuarios.ObtenerListadoPorNombre()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
          <Member Name="#ValidarPorId()" Kind="Method" Static="False" Accessibility="Public" ExternallyVisible="True">
           <Messages>
            <Message TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200" Status="Active" Created="2017-09-23 18:54:37Z" FixCategory="NonBreaking">
             <Issue Certainty="95" Level="Error" Path="c:\Users\Berry 1\Desktop\Proyectos Berry\ERP Almacen\Fuentes Modificados\Principal\Entidades" File="Usuarios.cs" Line="189">'Usuarios.ValidarPorId()' vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Issue>
            </Message>
           </Messages>
          </Member>
         </Members>
        </Type>
       </Types>
      </Namespace>
     </Namespaces>
    </Module>
   </Modules>
  </Target>
 </Targets>
 <Rules>
  <Rule TypeName="RethrowToPreserveStackDetails" Category="Microsoft.Usage" CheckId="CA2200">
   <Name>Reiniciar para mantener los detalles de la pila</Name>
   <Description>Utilice la instrucción de reinicio IL para mantener los detalles de la pila original cuando vaya a volver a provocar la excepción detectada.</Description>
   <Resolution Name="Default">{0} vuelve a producir una excepción detectada y la especifica explícitamente como un argumento. Utilice 'throw' sin un argumento, para mantener la ubicación de la pila donde se provocó inicialmente la excepción.</Resolution>
   <Owner />
   <Url>http://msdn.microsoft.com/library/ms182363.aspx</Url>
   <Email>[none]</Email>
   <MessageLevel Certainty="95">Error</MessageLevel>
   <File Name="usagerules.dll" Version="12.0.0.0" />
  </Rule>
 </Rules>
 <Localized>
  <String Key="Category">Categoría</String>
  <String Key="Certainty">Certeza</String>
  <String Key="CollapseAll">Contraer todo</String>
  <String Key="CheckId">Comprobar id.</String>
  <String Key="Error">Error</String>
  <String Key="Errors">errores</String>
  <String Key="ExpandAll">Expandir todo</String>
  <String Key="Help">Ayuda</String>
  <String Key="Line">Línea</String>
  <String Key="Messages">mensajes</String>
  <String Key="LocationNotStoredInPdb">[Ubicación no almacenada en Pdb]</String>
  <String Key="Project">Proyecto</String>
  <String Key="Resolution">Resolución</String>
  <String Key="Rule">Regla</String>
  <String Key="RuleFile">Archivo de regla</String>
  <String Key="RuleDescription">Descripción de la regla</String>
  <String Key="Source">Origen</String>
  <String Key="Status">Estado</String>
  <String Key="Target">Círculos concéntricos</String>
  <String Key="Warning">Advertencia</String>
  <String Key="Warnings">advertencias</String>
  <String Key="ReportTitle">Informe de análisis del código</String>
 </Localized>
</FxCopReport>
